### ISSUE TITLE
added madraka

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/project-documentation#169]
@dan: this one should be in Apte too — please check.

**Comment by danbalogh** [comment]
This one is a bit hairier. There is a Madraka in Apte (aka Bhadraka, Prabhadraka, Viśuddhacarita), but that is only _almost_ identical to this one. Your pattern from Zoetmulder is:
–⏑⏑|–⏑–|⏑⏑⏑|–⏑–|⏑⏑⏑|⏑-⏑|⏑⏑⏑|⏓
bha-ra-na-ra-na-ja-na-ga
Apte's pattern is:
–⏑⏑|–⏑–|⏑⏑⏑|–⏑–|⏑⏑⏑|**-⏑-**|⏑⏑⏑|⏓
bha-ra-na-ra-na-**ra**-na-ga

Can someone check a Javanese stanza to make sure this is not a typo in Zoetmulder?
I've checked Zoetmulder, so I know it is not your typo.
And I've looked up Apte's source (Piṅgala's Chandaḥśāstra), which gives the same definition, and Halāyudha's commentary there cites a Sanskrit example too).
So unless it's Zoetmulder's typo, we have a divergence between the Javanese and the Indian tradition (at least that represented by Piṅgala) here.
I have not done anything to the XML for the time being.

**Comment by arlogriffiths** [comment]
your observation is spot on. comparison with an actual instance reveals that Zoetmulder's pattern is in error for the 6th gaṇa.

<80.1a> lvirnikanaṅ kasogatan i bāli kādhikaraṇan muvah kuṭi hañar, 
<80.1b> lāvan i pūrvanāgara muvah vihāra bahuṅ ādirajya kuturan, 
<80.1c> nə̄m tikanaṅ kabajradharan uttama ṅhiṅ i vihāra taṅ kavinayan, 
<80.1d> kīrṇa makādiṅ ārya dadi rājasanmata kuṭinya tan vicaritan.

Could you try to add a `<note>` to the `<bibl>` for Zoetmulder stating as much?

**Comment by danbalogh** [comment]
Done, though <note> is not permitted anywhere by the schema, so I've just added an XML comment.

**Comment by arlogriffiths** [comment]
thanks!

**Comment by arlogriffiths** [comment]
@ajaniak : can you allow `<note>` to `<bibl>` in the schema for prosodicPatterns, so that we can can comment on the respective biblioghraphy items?

**Comment by ajaniak** [comment]
Done

**Comment by arlogriffiths** [comment]
@danbalogh: can you now convert that xml comment (and any others that deserve being foregrounded) to `<note>`?

**Comment by danbalogh** [comment]
Done. Not sure how necessary it is to distinguish notes from comments if this is to be only an internal document, but useful if some sort of display is planned for this. If so, I suggest that we change to `<note>` for everything that is "author to end user" communication and keep only "author to author" communication in XML comments.
If so, then it would probably be best to convert Zaki's comments on various alternative names and patterns into notes as well, but `<note>` is not allowed outside `<bibl>` so it may be necessary to permit it anywhere within `<item>`.
One more thing that I think would be really useful is to permit @resp on these notes, so that notes e.g. by Zaki and me could be distinguished in the markup. For the time being I've just kept zapa and daba at the beginning of the note text, as it was in the comment.
@arlogriffiths , what do you think of these two suggestions?
@ajaniak , can you implement them if Arlo gives the green light for either or both?

**Comment by arlogriffiths** [comment]
Thanks a lot. We already have a display: [https://erc-dharma.github.io/output/display-prosody.html](url). I fully agree with your suggestions. I hope Axelle can carry them out.

**Comment by danbalogh** [comment]
Lovely. For the display, we might also consider some sectioning for better overview.
E.g. nested `<div>`s with `<head>` for e.g.
Syllabo-quantitative metres
  1 syllable (class name)
  2 syllables (class name)
  etc.
Moraic metres
  āryā
  etc.
Miscellaneous
  to group vaitālīya types and perhaps generic templates such as pādānuṣṭubh and Vedic trimeter

**Comment by arlogriffiths** [comment]
Excellent suggestions. Can you apply the needed structure or tell Axelle which elements you think you'll need, if there use is currently not yet allowed?

**Comment by ajaniak** [comment]
`<note>` is already available everywhere in `<item>`. I have only kept its restriction regarding its use with `<bibl>`. 
@resp has been made available.

**Comment by ajaniak** [comment]
The first solution is fine.

**Comment by danbalogh** [comment]
I've created the sections. In addition to the three proposed above, I've made a fourth for "Additions in need of processing", containing the items whose comments say additions from Zaki and additions from Andrea.

Regarding display: I see that notes are now displayed at the end of the file. However, note anchors do not seem to be displayed (at least not in Firefox under Windows), so when someone looks at the table, they cannot know that there is a note attached at a certain point. I hope this can be solved. In addition, now that the list is sectioned, it may be best to put the notes pertaining to each table at the end of that table, rather than together at the end of the document as a whole.
